tox.ini

```diff
@@ -1,19 +1,30 @@
 [tox]
 minversion=2.3.1
 envlist =
-    py{27,35}-ansible22-{pylint,unit,flake8,yamllint,generate_validation}
+    py{27,35}-ansible22-lint
+    py{27,35}-ansible22-unit
 skipsdist=True
 skip_missing_interpreters=True
 
 [testenv]
 deps =
+    wheel
```

Comment:
Hmm, this one is tricky. In several cases this dependency is already installed with `pip` / `virtualenv`:
https://packaging.python.org/current/#id4

So in many cases adding it to `requirements.txt` (or here) is a no-op.

I don't have a strong case for it. We can see at the build log that it is a no-op in Travis:
https://travis-ci.org/openshift/openshift-ansible/jobs/202003785#L167